docs/queries/byrole.mdx

```diff
@@ -314,3 +315,36 @@ To learn more about the `aria-level` property, see
 
 > The `level` option is _only_ applicable to the `heading` role. An error will
 > be thrown when used with any other role.
+
+### `description`
+
+You can filter the returned elements by their
+[accessible description](https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_description)
+for those cases where you have several elements with the same role and they
+don't have an accessible name but they do have a description.<br /> This would
+be the case for elements with
+[alertdialog](https://www.w3.org/TR/wai-aria-1.1/#alertdialog) role, where the
+`aria-describedby` attribute is used to describe the elements content.
```

Comment:
From a spellchecker:

```suggestion
`aria-describedby` attribute is used to describe the element's content.
```

or 

```suggestion
`aria-describedby` attribute is used to describe the content of the element.
```

Reply:
Thanks for the catch 👍 

Updated in this commit: 9da0a1598132cb4c75ffa91d03ab9f16f24a9784